recipe/meta.yaml

```diff
@@ -1,13 +1,13 @@
 {% set name = "snowflake-snowpark-python" %}
-{% set version = "0.8.0" %}
+{% set version = "0.9.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://github.com/snowflakedb/snowpark-python/archive/refs/tags/v{{ version }}.tar.gz
-  sha256: a17d3f58f904937583a1a2d1b78aec4f215ed051c1d6043f9f0448e943dfa206
+  sha256: ebf65ab300b691189bc2972f5a81ab635dc91b8cf87c57a1c4f1eb09c409039d
 
 build:
   number: 0
```

Comment:
hmm <38 or >=39 ... should be simplified to py!=38 ? :)

Reply:
Additionally win64 builder seems to not run right now.  win32 isn't built by prefect anymore, but win64 should

Reply:
Oh good catch. I had to do a partial rebuild at one point

Reply:
No, it actually is python 3.8 only at this point

Reply:
@katietz change made! confirmed it is python 3.8 only:
https://github.com/snowflakedb/snowpark-python/blob/main/setup.py#L45